### PR TITLE
Add Occlum deployment guide for Azure AKS

### PR DESCRIPTION
**Pull request by jessehui**

### CODE REVIEW

**Comment by agowdamsft** [comment]
Couple of recommendations. 

1. Consider adding this https://docs.microsoft.com/en-us/azure/virtual-machines/dcv2-series link to the support SKU's where you mention EPC memory allocation for apps.
2. Mention SGX DCAP driver installation and Intel FSGS Patch installation on the confidential AKS nodes is already included (if required)
3. Worth mentioning Azure also provides related confidential services that customers can take benefits to have end to end protection for example Azure Attestation - https://aka.ms/azureattestation

**Comment by jessehui** [comment]
All comments are addressed.

File: README.md [hunk 1]
```diff
@@ -259,6 +259,10 @@ $ OCCLUM_RELEASE_ENCLAVE=yes occlum run <prog_path> <prog_args>
 
 Ultimately, whether an enclave is running in the release mode should be checked and judged by a trusted client through remotely attesting the enclave. See the remote attestation demo [here](demos/remote_attestation).
 
+## How to Run Occlum on Public Cloud?
+
+To cut off the complexity of self-hosted infrastructure, Occlum can also be easily deployed on public clouds with Confidential Computing resources. For example, we have tested and successfully deployed Kubernetes Pods on [Azure Kubernetes Service (AKS)](https://azure.microsoft.com/en-us/services/kubernetes-service/#getting-started). Please checkout [this doc](docs/azure_aks_deployment_guide.md) for more details.
```
Comment (tatetian) [inline README.md:264]: Some writing suggestions:
```
To cut off the complexity of self-hosted infrastructure, one can deploy Occlum-powered SGX apps on public clouds with SGX support. For example, we have tested and successfully deployed Occlum Docker containers on [Azure Kubernetes Service (AKS)](https://azure.microsoft.com/en-us/services/kubernetes-service/#getting-started). Please check out [this doc](docs/azure_aks_deployment_guide.md) for more details.
```

Comment (jessehui) [reply README.md]: Done. Thanks for your review.

**Pull request merged**